### ISSUE TITLE
Block Image in DOCX breaks the transformation of figures: docxtotei

### ISSUE THREAD

**Issue by Cutuchiqueno** [TEIC/Stylesheets#344]
In Word (up-to-date Word 365) the Layout-Option 'With Linebrake' -> 'Top and Bottom' (loosely translated from German) breaks sane transformation of figures -- and probably also of other objects that are able to use this option:

```
<figure>
    <p>
        <graphic n="1005" width="15.444611111111112cm" height="7.348361111111111cm" url="media/image3.png" rend="block"/>
        <hi style="color:red; font-size: 14pt; font-weight:bold;" rend="ERROR">�</hi>
        <note place="margin" type="conversion" resp="#teitodocx">
        <hi rend="docxError">unable to handle picture here, no embed or link</hi></note>
        Figure : Wortartenunterteilung nach morphologischen Kriterien (vgl. Pittner &amp; Berman 2004: 15)
    </p>
</figure>
```
Changing the Layoutoption to an Inline-Layout produces the expected result:

```
<figure>
    <graphic n="1006" width="15.444611111111112cm" height="7.348361111111111cm" url="media/image3.png" rend="inline">
        <desc>Ein Bild, das Gebäude, draußen, sitzend, alt enthält. generierte 
    <head>Wortartenunterteilung nach morphologischen Kriterien (vgl. Pittner &amp; Berman 2004: 15)</head>
</figure>

```
It seems to me that there is 1 issue in Word and 2 in the the stylesheets cause this result.

1. If you choose the abovementioned option the caption is encoded as a floating textblock with its own `w:p` inside the `w:p` of the image.
2. This somehow causes `tei:isCaption` or the `figureSection` template to overlook the caption as a caption of the image
3. Unfortunately, one of the rules in `tei:isFigure` is now again matching the `w:p` of the caption inside the `w:p` of the image, trying to find another image, but failing for obvious reasons so that the ERROR is produced.

It tried to track it down quick and dirty so the explanation might be wrong. As an ANT task it is unfortunately not possible to debug properly within Oxygen but I think something like this is has to be.

**Comment by sydb** [comment]
@martindholmes and I are trying to figure out this problem (pardon the pun). We cannot reproduce the problem for lack of access to a sufficiently modern enough version of MS Word. Is there any chance that you (@Cutuchiqueno) could provide two .docx files, one that has this problem and one that doesn’t? (Or one file with two figures, one that works, one that doesn’t.)

**Comment by Cutuchiqueno** [comment]
Yes I'll do next time when I get back to it my transformation tasks

**Comment by peterstadler** [comment]
@Cutuchiqueno any updates on this? We would need those two files for debugging …

**Comment by Cutuchiqueno** [comment]
Sorry, it completely got out of my way. I try to come back to this next week and provide the necessary resoruces

**Comment by martindholmes** [comment]
No response, closing.
